perf/h.js

```diff
@@ -0,0 +1,26 @@
+import bench from 'nanobench'
+import { h } from '../src/h'
+
+bench("vnode with no child 1M times", b => {
+  b.start()
+  for (let i = 0; i < 10000000; i++) {
```

Comment:
Would making a helper function for the 1M `for` loop be a bad idea in terms of the benchmark?

like:

```javascript
const ONE_MILLION = 1000000

function benchLoop(fn) {
  for (let i = 0; i < ONE_MILLION; i++) {
    fn()
  }
}
```

that way we can just do:

```javascript
b.start()
benchLoop(() => h("div", {}))
b.end()
```

Or am I just being picky with *DRY* 😂